app/Services/BulkEnrichmentService.php

```
@@ -364,6 +365,28 @@ public function enrichBook(int $bookId): array
             }
             $updateStmt->close();
 
+            // REG-1 (review): if we just populated a series name, sync the
+            // collane / libri_collane rows so admin views and detail-page
+            // hierarchy queries find the book. Pre-fix, bulk-enrich landed
+            // orphan series invisible to /admin/collane.
+            if (in_array('collana', $fieldsUpdated, true) && !empty($series)) {
+                try {
+                    (new \App\Models\SeriesRepository($this->db))->syncBookMemberships(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '374', column '26'). (undefined)

(MissingImport)